src/main/java/seedu/address/model/person/Birthday.java

```diff
@@ -0,0 +1,50 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.commons.util.AppUtil.checkArgument;
+
+public class Birthday {
+    public static final String MESSAGE_CONSTRAINTS =
+            "Birthdays should be in the form DD-MM-YYYY, and it should not be blank";
+
+    public static final String VALIDATION_REGEX = "^[0-9]{2}-[0-9]{2}-[0-9]{4}$";
+
+    public final String birthday;
+
```

Comment:
I think better to store as `LocalDate`. Can use #41 `dateUtil` to make it easy.

Reply:
Test cases seems to make it hard for you to pass in a LocalDate as an argument. I changed the validation to use #41 `dateUtil`. Should we still make it as a LocalDate field?

Reply:
Yes, we should still do it. I added some comments to the test files. Essentially, when we create the test objects, we parse the `dateStr` to a `LocalDate` using the util methods. i.e. `withBirthday(...)` takes in a `dateStr`, parses that string and returns a new `Birthday` with a `LocalDate` in it.